guides/typography.md

```diff
@@ -33,137 +36,72 @@ requests][font-inlining].
+| ------------------- | -------------------------------------------------------------------- |
+| `plain-family`      | [Optional] The font family to use for plain text, such as body text. |
+| `brand-family`      | [Optional] The font family to use for brand text, such as headlines. |
+| `bold-weight`       | [Optional] The font weight to use for bold text.                     |
+| `medium-weight`     | [Optional] The font weight to use for medium text.                   |
+| `regular-weight`    | [Optional] The font weight to use for regular text.                  |
 
-### Typography configs and theming
+These are used to generate the styles for the different typescale levels.
 
-You can provide a typography config when defining a theme to customize typographic styles. See the [theming guide][theming-system] for details on custom themes.
+## Type scale levels
 
-The following example shows a typical theme definition and a "kids theme" that only applies when
-the `".kids-theme"` CSS class is present. You can [see the theming guide for more guidance on
-defining multiple themes](https://material.angular.io/guide/theming#defining-multiple-themes).
+A **type scale** is a selection of font styles that can be used across an app.
+They’re assigned based on use (such as display or headline), and grouped more
+broadly into categories based on scale (such as large or small). For more
+information, see the [M3 typography spec](https://m3.material.io/styles/typography/type-scale-tokens).
 
-```scss
-@use '@angular/material' as mat;
+There are `large`, `medium`, and `small` variations for the following type roles:
+- **Display**: Styles are reserved for short, important text or numerals. They work best on large screens.
+- **Headline**: Headlines are best-suited for short, high-emphasis text on smaller screens. These styles can be good for marking primary passages of text or important regions of content.
+- **Title**: Titles are smaller than headline styles, and should be used for medium-emphasis text that remains relatively short. For example, consider using title styles to divide secondary passages of text or secondary regions of content.
+- **Body**: Body styles are used for longer passages of text in your app.
+- **Label**: Label styles are smaller, utilitarian styles, used for things like the text inside components or for very small text in the content body, such as captions.
 
-@include mat.core();
+The table below lists the CSS classes emitted and the native elements styled.
 
-$my-primary: mat.define-palette(mat.$indigo-palette, 500);
-$my-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
-$my-typography: mat.define-typography-config();
+| CSS class                 | Typesale level      | Native elements |
+|---------------------------|---------------------|-----------------|
+| `.mat-display-large`      | `display-large`     | `<h1>`          |
```

Comment:
I'm confused what the `Native` column means. Is this referring to the `typography-hierarchy` mixin applying styles to the native element?

Reply:
I meant to remove that (was there from the chart before), thanks for catching it!